core/tooltip.js

```diff
@@ -39,10 +40,28 @@ exports.TipInfo = TipInfo;
 
 /**
  * Is a tooltip currently showing?
+ * @type {boolean}
  */
 let visible = false;
+/** @deprecated September 2021 */
 exports.visible = visible;
```

Comment:
This is being overwritten by a call to `Object.defineProperties` below, and somehow I dislike that, though I can't articulate exactly why.

It this the easiest way to get the correct documentation and type info associated with the exported name?  I think I'd prefer a pure declaration, even though we'd be repeating the JSDoc:

```JS
/**
 * Is a tooltip currently showing?
 * @type {boolean}
 * @deprecated September 2021
 */
exports.visible;
```
Or even—since it's deprecated and a compiler warning might encourage people to migrate—just:

```JS
/** @deprecated September 2021 */
exports.visible;
```

("This is not important; wontfix" might be reasonable answer.)

Reply:
Done!